Add Anthropic integration (#33952)
home-assistant/home-assistant.github.io: source/_integrations/anthropic.markdown

---
title: Anthropic Conversation
description: Instructions on how to integrate Anthropic Claude as a conversation agent
ha_category:
  - Voice

> ha_release: 2024.8

Reviewer: This can probably be changed to the next HA release now.
```suggestion
ha_release: 2024.9
```

ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@Shulyaka'
ha_domain: anthropic
ha_integration_type: service
ha_platforms:
  - conversation
related:
  - docs: /voice_control/voice_remote_expose_devices/
    title: Exposing entities to Assist
  - docs: /voice_control/assist_create_open_ai_personality/
    title: Create an AI personality
  - url: https://console.anthropic.com/settings/keys
    title: Anthropic API key
  - url: https://www.anthropic.com
    title: Anthropic
  - url: https://claude.ai
    title: Claude
---

> The Anthropic integration adds a conversation agent powered by [Anthropic](https://www.anthropic.com), such as Claude 3.5 Sonnet, in Home Assistant.

Reviewer 2: ```suggestion
The **Anthropic** {% term integrations %} adds a conversation agent powered by [Anthropic](https://www.anthropic.com), such as Claude 3.5 Sonnet, in Home Assistant.
```

> Legal note: Anthropic currently limits the API usage to organizations only, more info here: [Can I use the Claude API for individual use?](https://support.anthropic.com/en/articles/8987200-can-i-use-the-claude-api-for-individual-use)

Reviewer 2: ```suggestion
```

Controlling Home Assistant is done by providing the AI access to the Assist API of Home Assistant. You can control what devices and entities it can access from the {% my voice_assistants title="exposed entities page" %}. The AI can provide you information about your devices and control them.

> 

Reviewer 2: ```suggestion

Legal note: Anthropic currently limits the API usage to organizations only, more info here: [Can I use the Claude API for individual use?](https://support.anthropic.com/en/articles/8987200-can-i-use-the-claude-api-for-individual-use)

```

This integration does not integrate with [sentence triggers](/docs/automation/trigger/#sentence-trigger).

> This integration requires an API key to use, [which you can generate here.](https://console.anthropic.com/settings/keys). This is a paid service, we advise you to monitor your costs in the [Anthropic portal](https://console.anthropic.com/settings/cost) closely.

Reviewer 2: ```suggestion

## Prerequisites

- This integration requires an API key to use, [which you can generate here.](https://console.anthropic.com/settings/keys). 
- This is a paid service, we advise you to monitor your costs in the [Anthropic portal](https://console.anthropic.com/settings/cost) closely.
```

> 
> {% include integrations/config_flow.md %}

Reviewer 2: ```suggestion
```

> ## Generate an API Key

Reviewer 2: ```suggestion
### Generating an API Key
```

The Anthropic API key is used to authenticate requests to the Anthropic API. To generate an API key, take the following steps:

> - Log in to the [Anthropic portal](https://console.anthropic.com) or sign up for an account.
> - Enable billing with a valid credit card on the [plans page](https://console.anthropic.com/settings/plans).
> - Visit the [API Keys page](https://console.anthropic.com/settings/keys) to retrieve the API key you'll use to configure the integration.

Reviewer 2: ```suggestion
1. Log in to the [Anthropic portal](https://console.anthropic.com) or sign up for an account.
2. Enable billing with a valid credit card on the [plans page](https://console.anthropic.com/settings/plans).
3. Visit the [API Keys page](https://console.anthropic.com/settings/keys) to retrieve the API key you'll use to configure the integration.
```

> 

Reviewer 2: ```suggestion

{% include integrations/config_flow.md %}

```

---
title: Anthropic Conversation
description: Instructions on how to integrate Anthropic Claude as a conversation agent
ha_category:
  - Voice
ha_release: 2024.9
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@Shulyaka'
ha_domain: anthropic
ha_integration_type: service
ha_platforms:
  - conversation
related:
  - docs: /voice_control/voice_remote_expose_devices/
    title: Exposing entities to Assist
  - docs: /voice_control/assist_create_open_ai_personality/
    title: Create an AI personality
  - url: https://console.anthropic.com/settings/keys
    title: Anthropic API key
  - url: https://www.anthropic.com
    title: Anthropic
  - url: https://claude.ai
    title: Claude
---

The **Anthropic** {% term integrations %} adds a conversation agent powered by [Anthropic](https://www.anthropic.com), such as Claude 3.5 Sonnet, in Home Assistant.

> 

Reviewer 2: ```suggestion
```

Controlling Home Assistant is done by providing the AI access to the Assist API of Home Assistant. You can control what devices and entities it can access from the {% my voice_assistants title="exposed entities page" %}. The AI can provide you information about your devices and control them.

Legal note: Anthropic currently limits the API usage to organizations only, more info here: [Can I use the Claude API for individual use?](https://support.anthropic.com/en/articles/8987200-can-i-use-the-claude-api-for-individual-use)

This integration does not integrate with [sentence triggers](/docs/automation/trigger/#sentence-trigger).

> 

Reviewer 2: ```suggestion
```
